### PR TITLE
Debugging & Grouping refactor

### PR DESCRIPTION
**Pull request by dacbd**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,12 +2,13 @@ const core = require('@actions/core');
 const github = require('@actions/github');
 
 async function run() {
-  console.log(github.context.payload);
+  core.debug(JSON.stringify(github.context.payload, null, 2));
```
Comment (geoffchan23) [inline index.js:5]: I used JSON stringify for something else and it complained about needing some package I think? That's why I was using console logs. I didn't realize there was a `core.debug`, but I'll start using that!

Comment (dacbd) [reply index.js:5]: the caveat that to see `core.debug` you have to run the action in debug mode

Comment (dacbd) [reply index.js:5]: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging

Comment (dacbd) [reply index.js:5]: Also the start/end group provides some helpful context/logging info more than a vinalla console.log at times. Particularly when there are distinct chunks of different behavior that are grouped together.

**Review by geoffchan23 — APPROVED** [review]

**Pull request merged**